DeepSpeech.py

```diff
@@ -425,6 +425,15 @@ def train():
 
     initializer = tf.global_variables_initializer()
 
+    # Disable progress logging if needed
+    if FLAGS.show_progressbar:
+        pbar_class = progressbar.ProgressBar
+        def log_progress(*args, **kwargs):
+            pass
+    else:
+        pbar_class = progressbar.NullBar
+        log_progress = log_info
```

Comment:
I think having some "progress and log class" in `utils` (that deals with the `show_progressbar` flag) would be much nicer. The additional code wouldn't hurt. Especially as it could get reused in `evaluate.py`.

Reply:
Done in 2351bf1